### PR TITLE
Revert "Merge pull request #1017 from david-salinas/remove_rocdl"

### PR DESCRIPTION
**Pull request by david-salinas**
This reverts commit baef698a0d39b85967e608a32b0f6e061d7369c9, reversing
changes made to e89cd3077924b7952da836b9b90657718d452e34.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -450,7 +544,7 @@ set(OFFICIAL_RELEASE_BUILD 1)
 # Default to generate a package appropriate for a distro; users override using command line with -D
 # -D writes variables into the cmake cache, and setting CACHE variables won't overwrite variables already in the CACHE
 if( "${DISTRO_ID}" MATCHES "ubuntu" )
-  set( CPACK_GENERATOR "DEB" CACHE STRING "cpack list: 7Z, DEB, IFW, NSIS, NSIS64, RPM, STGZ, TBZ2, TGZ, TXZ, TZ, ZIP" )
+  set( CPACK_GENERATOR "DEB" CACHE STRING "cpack list: 7Z, DEB, IFW, NSIS, NSIS64, RPM, STGZ, TBZ2, TGZ, TXZ, TZ, ZIP" FORCE )
 elseif ("${DISTRO_ID}" MATCHES "fedora" OR
         "${DISTRO_ID}" MATCHES "centos" OR
         "${DISTRO_ID}" MATCHES "redhatenterpriseserver" OR
```
Comment (scchan) [inline CMakeLists.txt:550]: Missing "FORCE" for RPM and TGZ

**Review by scchan — CHANGES_REQUESTED** [review]

File: CMakeLists.txt (hunk 1 above)
Comment (david-salinas) [reply CMakeLists.txt:550]: Maybe I don't understand the syntax here.  Isn't putting FORCE at the end of the string "" what we need here?

Comment (scchan) [reply CMakeLists.txt:550]: I mean you need to add "FORCE" to line 458 and 461 as well

Comment (david-salinas) [reply CMakeLists.txt:550]: ah ok.  I just pushed an update

**Review by scchan — APPROVED** [review]

**Comment by whchung** [comment]
Since this is a pretty big retrogression, may I understand the story behind this decision?

I'm actually fine either way (rocm-device-libs shipped inside HCC, or a standalone package). But we do need one particular commit in rocm-device-libs in ROCm 2.3 no matter what ( https://github.com/RadeonOpenCompute/ROCm-Device-Libs/commit/7aca7eeb441ece8eff839f0fa8a5bb1682b74897 ) . Would it be possible to make sure rocdl submodule within HCC to have this commit?

**Comment by david-salinas** [comment]
@whchung  HCC will pick up latest rocdl from master.  It looks like that commit was merged a couple of weeks ago, so it should be in HCC's build of device-libs.

**Pull request merged**